modules/user_accounts/php/edit_user.class.inc

```diff
@@ -1200,6 +1200,7 @@ class Edit_User extends \NDB_Form
         if (isset($values['NA_Password'])
             && $values['NA_Password'] == "on"
             && $values['SendEmail'] != "on"
+            && empty($this->identifier)
```

Comment:
I dont think this is a good idea. even if the user has been created without a password (i.e. from request account) you have to validate that they are emailed when a password is generated

Reply:
Good catch! this is not a bug, the user pass is null. I will close this PR.